src/resource.rs

```diff
@@ -0,0 +1,49 @@
+use sdl2::ttf::{Sdl2TtfContext, Font};
+use sdl2;
+use std::path::PathBuf;
+use std;
+
+pub struct ResourceLoader {
+    res_prefix: String,
+    res_dir: PathBuf,
+    ttf: Sdl2TtfContext,
+}
+
+impl ResourceLoader {
```

Comment:
I'd suggest writing some tests for resource parsing

Reply:
Agreed. Will create separate issue for that.